cromwell_tools/cli.py

```diff
@@ -40,7 +40,7 @@ def parser(arguments=None):
                        help='Cromwell password for HTTPBasicAuth.')
         p.add_argument('--secrets-file', dest='secrets_file', default=None, type=str,
                        help='Path to the JSON file containing username, password, and url fields.')
-        p.add_argument('--service_account_key', default=None, type=str,
+        p.add_argument('--service-account-key', dest='service_account_key', default=None, type=str,
```

Comment:
@rexwangcc @samanehsan do you think it'd be worth the complication to try centralizing these field names into a single variable, to avoid future mismatches? Something like:
```python
arg = 'the_key'
def to_opt(key_name):
  # Probably not actual python...
  return '--%s' % key_name.replaceAll('_', '-')
p.add_argument(to_opt(arg), dest=arg, type=str, ...)
# Farther below...
auth_args = [..., arg, ...]
```

Reply:
It sounds like a good idea to me! Furthermore, I'd like to find a way to map APIs to CLIs programmatically so we(I) don't make stupid mistakes like this anymore!